src/repos.json

```diff
@@ -458,6 +458,15 @@
       "forks": 4,
       "description": "Razorpay Gravity Forms plugin",
       "language": "PHP"
+    },
+    {
+      "html_url": "https://github.com/razorpay/kubestash-v2",
+      "name": "kubestash-v2",
+      "categories": ["kubernetes", "credstash"],
```

Comment:
Can you capitalize the first character(`k` and `c`) here?

Reply:
done